### PR TITLE
APEXMALHAR-1982: CassandraInputOperator query setter fails with null …

### PR DESCRIPTION
**Pull request by DT-Priyanka**
…pointer exception

### CODE REVIEW

**Comment by DT-Priyanka** [comment]
@chandnisingh  please review.

File: contrib/src/test/java/com/datatorrent/contrib/cassandra/CassandraOperatorTest.java [hunk 1]
```diff
@@ -293,8 +293,8 @@ public void testCassandraInputOperator()
 
     TestInputOperator inputOperator = new TestInputOperator();
     inputOperator.setStore(store);
-    inputOperator.setTablename(TABLE_NAME_INPUT);
     inputOperator.setQuery(query1);
+    inputOperator.setTablename(TABLE_NAME_INPUT);
```
Comment (bhupeshchawda) [inline contrib/src/test/java/com/datatorrent/contrib/cassandra/CassandraOperatorTest.java:297]: Please comment on why this change was needed, or undo this change as it is just reordering of setters on the operator

Comment (DT-Priyanka) [inline contrib/src/test/java/com/datatorrent/contrib/cassandra/CassandraOperatorTest.java:297]: Previously, query was dependent on "tableName" variable. As the order of setting variables was tableName followed by query here the test case always passed. To simulate situation where platform could set tablename after query (there is no guarantee in which order variables are set) I did this change.

**Comment by bhupeshchawda** [comment]
Change looks good. Will merge this if there are no other comments.
